src/asRendition.tsx

```diff
@@ -60,32 +55,26 @@ export const withStyledSystem = (child: React.StatelessComponent) => {
 	// @ts-ignore
 	Base.propTypes = propTypes;
 
-	return (props: any) => {
-		return <Base {...props} />;
-	};
+	return React.forwardRef((props: any, ref: any) => {
+		return <Base {...props} ref={ref} />;
+	});
 };
 
-// This middleware allows component to fallback to the default theme when
-// rendered without a ThemeProvider wrappwer
-export const withDefaultTheme = (
-	Base: React.ComponentClass | React.StatelessComponent,
-) => {
-	if (!Base.defaultProps) {
-		Base.defaultProps = {};
-	}
-
-	(Base.defaultProps as any).theme = theme;
-
-	return (props: any) => {
-		return <Base {...props} />;
-	};
-};
+const enhancers = [withTheme, withTooltip, withStyledSystem];
 
-const Tag = tag(blacklist);
```

Comment:
How come this has been removed?

Reply:
It isn't properly implemented (doesn't forward `ref`), so I temporarily removed it and will add a new hoc (it's a 3-liner anyway, no need to depend on another package)